scw_serverless/config/route.py

```diff
@@ -33,12 +33,3 @@ def validate(self) -> None:
         for method in self.http_methods or []:
             if method not in HTTPMethod:
                 raise RuntimeError(f"Route contains invalid method {method.value}")
-
-    def asdict(self) -> dict[str, Any]:
```

Comment:
This only is highly dependent of the gateway, so I prefer to have it in the gateway part of the code. It's what we do the other config options.